packages/web-components/fast-components/src/select/index.ts

```diff
@@ -21,22 +39,20 @@ export const fastSelect = Select.compose<SelectOptions>({
     styles,
     indicator: `
         <svg
-            class="select-indicator"
-            part="select-indicator"
-            viewBox="0 0 12 7"
+            width="16"
+            height="16"
+            viewBox="0 0 16 16"
             xmlns="http://www.w3.org/2000/svg"
         >
             <path
-                d="M11.85.65c.2.2.2.5 0 .7L6.4 6.84a.55.55 0 01-.78 0L.14 1.35a.5.5 0 11.71-.7L6 5.8 11.15.65c.2-.2.5-.2.7 0z"
+                d="M3.44 6.72a.83.83 0 0 1 .63-1.39h7.86c.72 0 1.1.85.63 1.39l-3.68 4.2c-.47.53-1.3.53-1.76 0l-3.68-4.2Zm1-.39 3.43 3.93c.07.08.19.08.26 0l3.43-3.93H4.44Z"
```

Comment:
I cleaned up this icon in Figma. Please export and optimize and replace.

Reply:
updated